### PR TITLE
fix: disable mutation of typescript ast in converter

### PR DESCRIPTION
**Pull request by armano2**
This PR fixes warning from ts:

>Warning From TSC: "Debug Failure. Shouldn't ever directly check a JsxOpeningElement

### CODE REVIEW

File: src/convert.ts [hunk 1]
```diff
@@ -2080,22 +2080,22 @@ export default function convert(config: ConvertConfig): ESTreeNode | null {
...
-       */
-      (node as any).kind = SyntaxKind.JsxOpeningElement;
-
-      const openingElement = convertChild(node);
-      (openingElement as any).selfClosing = true;
-
       Object.assign(result, {
         type: AST_NODE_TYPES.JSXElement,
-        openingElement,
+        openingElement: {
```
Comment (JamesHenry) [inline src/convert.ts]: Please can we keep the original comment and just move it to here instead?

**Comment by JamesHenry** [comment]
Thanks for patching this so quickly!

File: src/convert.ts (hunk 1 above)
Comment (armano2) [reply src/convert.ts]: ok

**Comment by armano2** [comment]
i don't like warnings in my console xd

**Pull request merged**